### ISSUE TITLE
java.lang.ClassCastException: class soot.jimple.NullConstant cannot be cast to class soot.Local 

### ISSUE THREAD

**Issue by zosrothko** [adoptium/emt4j#38]
Hi
Got the below exception
```
Failed to analyze opennlp-tools-1.5.3.jar
java.lang.ClassCastException: class soot.jimple.NullConstant cannot be cast to class soot.Local (soot.jimple.NullConstant and soot.Local are in unnamed module of loader org.eclipse.emt4j.common.staticanalysis.StaticAnalysisEntry$SootAnalysisClassLoader @1c145458)
        at org.eclipse.emt4j.common.staticanalysis.impl.CastArraysAsListToArrayAnalyzer.doAnalyze(CastArraysAsListToArrayAnalyzer.java:66)
  Analyze bctsp-jdk14-1.38.jar done
        at org.eclipse.emt4j.common.staticanalysis.impl.BaseAnalyzer.analyze(BaseAnalyzer.java:45)
        at org.eclipse.emt4j.common.staticanalysis.impl.CastArraysAsListToArrayAnalyzer.analyze(CastArraysAsListToArrayAnalyzer.java:41)
        at org.eclipse.emt4j.common.staticanalysis.InvokerImpl.invoke(InvokerImpl.java:117)
        at org.eclipse.emt4j.common.staticanalysis.StaticAnalysisEntry.analyze(StaticAnalysisEntry.java:75)
        at org.eclipse.emt4j.common.rule.impl.WholeClassRule.check(WholeClassRule.java:68)
        at org.eclipse.emt4j.common.rule.ExecutableRule.execute(ExecutableRule.java:70)
        at org.eclipse.emt4j.analysis.AnalysisExecutor.lambda$null$1(AnalysisExecutor.java:99)
        at org.eclipse.emt4j.analysis.analyzer.ClassAnalyzer.processClass(ClassAnalyzer.java:65)
  Analyze javafx-swing-11.0.2-win.jar done      at org.eclipse.emt4j.analysis.analyzer.JarAnalyzer.analyze(JarAnalyzer.java:60)
        at org.eclipse.emt4j.analysis.analyzer.DependencyAnalyzer.iterateDo(DependencyAnalyzer.java:52)
        at org.eclipse.emt4j.analysis.source.SingleJarSource.parse(SingleJarSource.java:38)
        at org.eclipse.emt4j.analysis.AnalysisExecutor.lambda$execute$2(AnalysisExecutor.java:91)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654)
  Analyze izpack-api-5.2.0.jar done
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:290)

        at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

**Comment by D-D-H** [comment]
Thanks for reporting these errors, and I will try to fix them this week.

**Comment by D-D-H** [comment]
Hi, this problem has been fixed by dd607cabe05b31b9982661ef25e8d1b16e56e93d.
